App.js

```diff
@@ -0,0 +1,43 @@
+// get elements from html
+
+const openModal = document.querySelector('#mobile-menu');
```

Comment:
Kindly update `linters.yml` content with this [one](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) and add [.eslintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.eslintrc.json)  in order for javascript linters to work. For any javascript project use this [linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) instead of old linters that contains HTML and CSS only.

Reply:
I have added it